zboxcore/sdk/commitworker.go

```diff
@@ -283,15 +276,13 @@ func (commitreq *CommitRequest) calculateHashRequest(ctx context.Context, paths
 			return err
 		}
 		defer resp.Body.Close()
-		if resp.StatusCode != http.StatusOK {
```

Comment:
Are all non-OK responses contain a body?
I think that was the reason there were this early copy of the check.